apps/wallet/src/ui/app/shared/card/index.tsx

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import cl from 'classnames';
+import { memo } from 'react';
+
+import type { ReactNode } from 'react';
+
+export type CardProps = {
+    className?: string;
+    children?: ReactNode | ReactNode[];
+};
+
+function Card({ className, children }: CardProps) {
```

Comment:
I think base on the design, using props for the header footer will be a bit hairy. Since the will be multiple element inside each props

![Screenshot 2022-12-13 at 3 02 16 PM](https://user-images.githubusercontent.com/8676844/207433149-f4415441-d5c7-4414-a2a7-69b265116607.png)
![Screenshot 2022-12-13 at 3 03 38 PM](https://user-images.githubusercontent.com/8676844/207433151-d181df6a-8d8a-4b46-89e7-4512866e6e4c.png)

Reply:
I'm actually not convinced these should be the same component.

Reply:
But even so I actually think this design is perfectly compatible with using props for header and footer.

Reply:
the idea is to include `CoinBalance` component in the header of the Card. But point noted

Reply:
```tsx
function SplitList({ children }) {
  return <div className="grid grid-cols-2 divide-x">{children}</div>
}

// later...
<Card header={<SplitList><CoinBalance /><CoinBalance /></SplitList>}>
  Card Content Here
</Card>
```

I think the one benefit of header props vs functions is that you can guarantee there's only one in the component (plus better DX)